hosted/gke/helper/helper_cluster.go

```diff
@@ -227,3 +243,43 @@ type ImportClusterConfig struct {
 	NodePools []*management.GKENodePoolConfig `json:"nodePools" yaml:"nodePools"`
 	Labels    *map[string]string              `json:"labels,omitempty" yaml:"labels,omitempty"`
 }
+
+// DefaultGKE returns the default GKE version used by Rancher
+func DefaultGKE(client *rancher.Client, projectID, cloudCredentialID, zone, region string) (defaultGKE string, err error) {
```

Comment:
Same comment as above regarding common function

Reply:
This cannot be a common function because every endpoint returns a different response. AKS simply returns a list of k8s versions, some of which are still in preview mode. So we need to call another API endpoint `v3/settings/ui-k8s-default-version-range` to obtain the default version range and then calculate the defaultAKS version.